specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/examples/StorageAccountGetMigrationInProgress.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "resource-group-name",
+    "accountName": "{storage-account-name}",
+    "api-version": "2023-01-01",
+    "monitor": "true"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscription-id}/resourceGroups/resource-group-name/providers/Microsoft.Storage/storageAccounts/{storage-account-name}/accountMigrations/default",
+        "name": "default",
+        "type": "Microsoft.Storage/storageAccounts/accountMigrations",
+        "properties": {
+          "targetSkuName": "Standard_ZRS",
+          "migrationStatus": "InProgress�"
```

Comment:
It looks there's a special char in the line. 
If this is not by design, please fix.

Reply:
fixed